src/main/g8/src/main/scala/$package$/$classname$.scala

```diff
@@ -0,0 +1,8 @@
+package $package$
+
+import com.danielasfregola.twitter4s.TwitterRestClient
+
+object $classname$ extends App {
+  val client = TwitterRestClient()
```

Comment:
[Unrelated to this PR] Maybe we should have two mains: one for rest, one for streaming and perform a simple operation with the client...but if we decide to do this, no worries, we will do in a separate PR/issue